exec/templates/gnu.mk

```diff
@@ -96,9 +101,7 @@ else
 FPPFLAGS += $(NETCDF_FLAGS)
 endif
 # Fortran Compiler flags for the MPICH MPI library
-ifndef MPI_FLAGS
-FPPFLAGS += $(shell pkg-config --cflags-only-I mpich2-c)
-else
+ifdef MPI_FLAGS
```

Comment:
@underwoo did you intend to also change the MPI flags in the make templates?  It's not mentioned in the commit message.